### DISCUSSION TITLE
What are your thoughts on NvChad relying on null-ls which is abandoned?

### DISCUSSION THREAD

**Discussion by flatoy** [NvChad/NvChad#2504] [Q&A] [answered]
I just read [this post](https://github.com/jose-elias-alvarez/null-ls.nvim/issues/1621) from the maintainer of `null-ls` who abandoned it this summer.

**Comment by siduck** [comment, marked as answer]
@indiehackor use conform.nvim and nvim-lint

**Reply by flatoy** [reply]
okay, but null-ls will still ship with NvChad by default?

**Reply by siduck** [reply]
@indiehackor no, in example_config u mean? i'll replace it soon

**Comment by guzman109** [comment]
Use none-ls. It's a fork of null maintained by the community. But honestly, until null breaks on you, there's no reason to switch.
